### DISCUSSION TITLE
Ld2410 no MQTT measurement values

### DISCUSSION THREAD

**Discussion by NeigbourOfTheBeast** [arendst/Tasmota#18579] [Q&A]
Hi,
I have build the Binary with the ld4210 define.
Worked so far,  I'm able to see the values on the tasmota configuration screen, they look good so far. I did the mqtt configuration, it worked do far that I'm able to see the device on the server but only with the generic stuff like alive info and so on. There are no measurement values  what did I do wrong?

Thanks in advance for the support..

**Comment by barbudor** [comment]
Logs?
What server?

**Reply by sfromis** [reply]
What I see with LD2410 are MQTT payloads like:
`.../SENSOR = {"Time":"2023-05-05T02:27:25.962+02:00","LD2410":{"Distance":[58.0,0.0,0.0],"Energy":[0,63]}}`

Operation can be configured using these commands:
```
 * LD2410Duration 0                            - Set factory default settings
 * LD2410Duration 1..65535                     - Set no-one duration in seconds (default 5)
 * LD2410MovingSens 50,50,40,30,20,15,15,15,15 - Set moving distance sensitivity for up to 9 gates (at 0.75 meter interval)
 * LD2410StaticSens 0,0,40,40,30,30,20,20,20   - Set static distance sensitivity for up to 9 gates (at 0.75 meter interval)
 ```

https://github.com/arendst/Tasmota/blob/development/tasmota/tasmota_xsns_sensor/xsns_102_ld2410.ino#L10-L26

**Reply by NeigbourOfTheBeast** [reply]
The Mqtt output on the console seems to be ok.
But the MQTT Server there is no measurement data visible.
![Screenshot_20230505_182314_Chrome](https://user-images.githubusercontent.com/18751435/236513720-35a73cd4-cb87-4189-83c3-e74ded4619b2.jpg)


17:14:24.407 MQT: tele/tasmota_2F73C4/SENSOR = {"Time":"2023-05-05T17:14:24","LD2410":{"Distance":[30.0,0.0,0.0],"Energy":[100,100]}}

**Reply by barbudor** [reply]
So Tasmota is doing its job

What is this "server" you don't want to name ?
If it is HomeAssistant, you need to include `#define USE_TASMOTA_DISCOVERY` in your build and have the Tasmota plugin installed in HA

**Reply by sfromis** [reply]
Your screen dump looks like being from the wrong topic (STATE), instead of the SENSOR topic where the data is.

**Comment by NeigbourOfTheBeast** [comment]
The Server is an ipbroker sonoff instance

Barbudor ***@***.***> schrieb am Fr., 5. Mai 2023, 18:31:

**Reply by barbudor** [reply]
No idea what this is and even Google doesn't know ¯\_(ツ)_/¯
We can help with Tasmota but with other softwares

**Comment by NeigbourOfTheBeast** [comment]
Sorry was misspelled it's an iobroker Server, the sonoff Instance is acting
as MQTT server

Barbudor ***@***.***> schrieb am Fr., 5. Mai 2023, 19:00:

**Comment by NeigbourOfTheBeast** [comment]
That's the Catch there is no sensor topic. Usually an additional folder is
shown.

sfromis ***@***.***> schrieb am Fr., 5. Mai 2023, 18:43:

**Reply by sfromis** [reply]
What you posted shows that Tasmota absolutely is publishing to the SENSOR topic. If you are using [ioBroker.sonoff](https://github.com/ioBroker/ioBroker.sonoff/), I think that you may have to do some extra setup there around discovery, or manual configuration.

**Comment by NeigbourOfTheBeast** [comment]
Thanks for narrowing it down and the Quick feedback, I'll check the other
side

sfromis ***@***.***> schrieb am Fr., 5. Mai 2023, 19:50:

**Comment by NeigbourOfTheBeast** [comment]
The information might be helpful for someone in a similar situation:
Found the issue meanwhile, after activating the option "Sensor" and the "build tree" option
